src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java

```diff
@@ -1,12 +1,11 @@
 package seedu.address.model.person;
 
 import java.util.List;
+import java.util.Locale;
```

Comment:
Looks good, but you may want to rename this class in the future for greater clarity!